tripal/src/Plugin/TripalPubLibrary/TripalPubLibraryPubMed.php

```diff
@@ -443,6 +451,8 @@ private function pmidFetch(string $rettype, string $retmod, int $start,
 
     $results = $this->fileretriever->retrieveFileContents($fetch_url, $this->retrieval_options);
 
+    // If NCBI is down for maintenance a HTML page may be embedded in the results.
+    print "\nCP5:";var_dump($results);
```

Comment:
Should this still be in the final PR or is it debugging code?

Reply:
Oops - will remove

Reply:
Removed with commit https://github.com/tripal/tripal/pull/2246/commits/6a23cf862dbf90a4c0d5593622551b658c0b19c2